addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1620,6 +1620,53 @@ QUnit.test('chat window with a thread: keep scroll position in message list on f
     );
 });
 
+QUnit.test('chat window should scroll at the end after a new message is posted', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+        state: 'open',
+        is_pinned: true,
+    });
+    for (let i = 0; i < 10; i++) {
+        this.data['mail.message'].records.push({
+            channel_ids: [20],
+            model: 'mail.channel',
+            res_id: 20,
+        });
+    }
+    await this.start({
+        env: {
+            browser: {
+                innerWidth: 1920,
+            },
+        },
+    });
+
+    // Set content of the composer of the chat window
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, 'WOLOLO');
+    });
+    await afterNextRender(() =>
+        triggerEvent(
+            document.querySelector('.o_ChatWindow .o_ComposerTextInput_textarea'),
+            'keydown',
+            { key: 'Enter' },
+        )
+    );
+    /**
+     * To determine if an element has been totally scrolled or not
+     * @see https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight#Problems_and_solutions
+     */
```

Comment:
It's not the first time we do that assertion, I think it's fine without the comment.
Maybe put `document.querySelector('.o_MessageList')` into a `messageList` variable to shorten these lines a bit.